src/js/bandwidth_test.js

```diff
@@ -157,6 +158,14 @@ function testVideoBandwidth(config) {
         rttStats.add(Date.parse(report.timestamp),
           parseInt(report.stat('googRtt')));
       }
+      if (report.type === 'ssrc') {
+        // This prevents it from collecting the intial send resolution which is
+        // something like 176x1407.
+        if (videoStats.length < 20) {
+          videoStats[0] = parseInt(report.stat('googFrameWidthSent'));
+          videoStats[1] = parseInt(report.stat('googFrameHeightSent'));
+        }
```

Comment:
We could send different resolutions during a call, depending on cpu adaptation etc. This will show the last resolution, right? Probably makes sense. Or should we show the max?